perf.groovy

```diff
@@ -32,10 +38,8 @@ def generate(boolean isPr) {
     Utilities.addArchival(myJob, archiveSettings)
     Utilities.standardJobSetup(myJob, projectName, isPr, defaultBranch)
     Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-perf')
-    Utilities.addGithubPushTrigger(myJob)
```

Comment:
Why are we removing the push trigger?  Won't this block running perf when a commit happens?